src/doc/gui_manual/Operators/OperatorTypes/PICS/index.rst

```diff
@@ -0,0 +1,268 @@
+Parallel Integral Curve System
+==============================
+
+Within the VisIt infrastructure is the ability to generate integral curves:
+streamlines and pathlines. An integral curve is a curve that begins at a seed
+location X, and is tangent at every point in a vector field. An integral curve
+is computed by numerical integration of the seed location through the vector
+field. The image below shows fieldlines through the magnetic field of a
+core-collapse supernova simulation from the GenASiS code.
+
+IMAGE HERE
+
+The generation of the curves form the basis of VisIt's Parallel Integral Curve
+System (PICS), made up of the :ref:`Integral Curve operator`, the :ref:`LCS
+operator`, the :ref:`Limit Cycle operator`, and the :ref:`Poincare operator`.
+Much of the underlying infrastructure and interface is the same for each
+operator. The user selects a series of seed locations where curves are
+generated, which are then visualized and analyzed.
+
+The PICS allows for the computation of Lagrangian Coherent Structures (LCS)
```

Comment:
`The PICS` replace with either `PICS` or `The PICS system`

Reply:
I think it's okay the way it is because the S is already system and I think the "The" is good to have too because it read like "The Integral Curve System allows...."